docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -141,6 +143,33 @@ configuration). If the subsystem detects that the `OIDC` mechanism is indeed req
 activate the authentication mechanism automatically. Otherwise, no activation will occur and deployment
 will continue normally.
 
+=== Scope Configuration
+
+NOTE: Since this attribute is a community level feature, the WildFly server's stability level must be set to community
+level or lower to enable it. The stability level must not be changed to default after configuring the
+`scope` attribute. For more information about the stability levels of the WildFly server, please see
+https://docs.wildfly.org/31/Admin_Guide.html#Feature_stability_levels[here].
```

Comment:
Instead of hardcoding the URL here, you can specify it the way Diana did here:

https://github.com/wildfly/wildfly/pull/13687/commits/435340810b75956671aaa9cc9ca39e8c7bf2839b#diff-ff7d49f604470857104bc4dd96ab9358f3f857e61fa9d8c23990b1c0eb2f58d2R2675-R2676

That way we don't need to keep updating the link as the version number changes.

Reply:
fixed!